jenkins/ci.suse.de/pipelines/openstack-ardana-gerrit.Jenkinsfile

```diff
@@ -56,9 +54,13 @@ pipeline {
               string(name: 'tempest_run_filter', value: "$tempest_run_filter"),
               string(name: 'develproject', value: "$develproject"),
               string(name: 'repository', value: "$repository")
-            ], propagate: true, wait: true
-          } finally {
-            echo slaveJob.buildVariables.blue_ocean_buildurl
+          ], propagate: false, wait: true
+          def jobResult = slaveJob.getResult()
+          def jobUrl = slaveJob.buildVariables.blue_ocean_buildurl
+          def jobMsg = "Build ${jobUrl} completed with: ${jobResult}"
+          echo jobMsg
+          if (jobResult != 'SUCCESS') {
```

Comment:
I see that for the failure case the message is printed 2 times.
What if you move the `echo jobMsg` to after the if, or using `else`?

Reply:
I already tried that and ended up with an empty "Error signal" message in the Blue Ocean UI (https://ci.nue.suse.com/blue/organizations/jenkins/openstack-ardana/detail/openstack-ardana/974/pipeline )

Reply:
Thanks, in that case, looks good!